common/lib/xmodule/xmodule/tests/test_html_module.py

```diff
@@ -1,6 +1,12 @@
 import unittest
-import ddt
 from mock import Mock
+import ddt
+
+from django.test.utils import override_settings
+
+from opaque_keys.edx.locator import CourseLocator
+from xblock.field_data import DictFieldData
+from xblock.fields import ScopeIds
```

Comment:
@macornwell  Please remove these 3 lines to fix the CircleCI tests:
* `CourseLocator` is not used, and was flagged by pylint
* `DictFieldData` is imported below
* `ScopeIds` is imported below

Reply:
Fixed them.